capstone/src/main/java/com/google/sps/data/RequestJsonParser.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.data;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.io.BufferedReader;
+import com.google.gson.Gson;
+
+public class RequestJsonParser {
+
+    public static <T> T parseObjectFromRequest(HttpServletRequest request, Class<T> type) throws IOException {
+        BufferedReader reader = request.getReader();
+        StringBuilder sb = new StringBuilder();
+        try {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                sb.append(line).append('\n');
+            }
+        } finally {
+            reader.close();
+        }
```

Comment:
Cool Library. I will check it out

Reply:
After some discussion with Kyle, I think it'll be better to add this library in PR#8 since PR#7 has a different implementation of this and (I believe) there will be merge conflicts as we merge up.

Reply:
So for now, we'll leave this functionality of manually reading string builder with for loop until PR#8.